ci/conda_env_cpp.txt

```diff
@@ -31,16 +31,16 @@ gflags
 glog
 gmock>=1.10.0
 google-cloud-cpp>=1.34.0
-grpc-cpp<=1.50.1
 gtest>=1.10.0
+libgrpc
 libprotobuf
 libutf8proc
 lz4-c
 make
 meson
 ninja
 nodejs
-orc
+orc<2.1.0
```

Comment:
In CI orc is pinned to 1.9.0. This new pin is required due to some upstream bugs with the CMake configuration - see https://github.com/apache/orc/issues/2408